src/drivers/drv_sensor.h

```diff
@@ -84,6 +84,7 @@
 #define DRV_RNG_DEVTYPE_MB12XX   0x31
 #define DRV_RNG_DEVTYPE_LL40LS   0x32
 #define DRV_ACC_DEVTYPE_MPU6050  0x33
+#define DRV_IMU_DEVTYPE_ICM45686 0x34
```

Comment:
We should delete `DRV_ACC_DEVTYPE_MPU6050` here and move the DRV_RNG options to group them with DRV_DIST.

I'll do it in a followup unless you want the device id to be stable here.

Reply:
Yer, a follow up is fine.